index.html

```diff
@@ -5,31 +5,54 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="css/popup.css" />
     <title>Portfolio</title>
   </head>
   <body>
     <header id="header">
       <nav>
-        <a id="logo" href="index.html">Abdullah Asghar</a>
+        <a id="logo" href="index.html">Abdullah</a>
         <img src="./icon/Icon-Menu.png" class="menu" alt="Menu-icon" />
+        <div class="mobile-menu">
+          <img src="./images/Icon-Cancel.png" alt="close-icon"  id="close-btn">
+        </div>
       </nav>
-      <div class="navbar">
-        <ul id="desktop-nav-links">
-          <li>
-            <a href="#">Portfolio</a>
-          </li>
-          <li>
-            <a href="#">About</a>
-          </li>
-          <li>
-            <a href="#">Contact</a>
-          </li>
-          <li>
-            <a href="#">
-              <img src="./icon/Icon-Mail.png" class="mail" alt="mail icon" />
-            </a>
-          </li>
-        </ul>
+      <div class="mob-menu-hidden open-menu">
+        <div class="navbar">
+          <ul class ="menu-item desktop-nav-links">
+            <li>
+              <a href="#recent-works">Portfolio</a>
+            </li>
+            <li>
+              <a href="#about-section">About</a>
+            </li>
+            <li>
+              <a href="#contact-form">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </div>
```

Comment:
Your design doesn't quite match the template design.

Your design:

<img width="184" alt="Screen Shot 2022-06-14 at 12 50 35" src="https://user-images.githubusercontent.com/82426682/173655845-edbdb7b5-09e8-49c3-b449-700036e3c8bb.png">

The template design:

<img width="166" alt="Screen Shot 2022-06-14 at 12 51 04" src="https://user-images.githubusercontent.com/82426682/173656251-a0439bf0-3643-4fcb-9e2f-3e15fa831828.png">

Some (but not all) of the things you should fix are background color, font, font size, margins, padding, and dividers. Please make sure your mobile menu looks identical to the Figma template.

Reply:
Changes required done